src/transformers/models/llava/modeling_llava.py

```diff
@@ -270,22 +270,22 @@ def resize_token_embeddings(self, new_num_tokens: Optional[int] = None, pad_to_m
     def _merge_input_ids_with_image_features(
         self, image_features, inputs_embeds, input_ids, attention_mask, position_ids
     ):
-        nb_images, image_hidden_dim, embed_dim = image_features.shape
+        num_images, num_image_patches, embed_dim = image_features.shape
```

Comment:
this is not really a "hidden dim" but rather the number of image patches. As below "num_image_tokens" is already used, this could be confusing with "num_image_patches". The former actually refers to the number of special image tokens, hence I've renamed that.